azure-pipelines-v3.yml

```diff
@@ -18,63 +18,6 @@ variables:
 
 jobs:
 
-# Test pull requests on linux and mac
-- job: LinuxBuild
-  pool:
-    vmImage: 'ubuntu-latest'
-  condition: eq(variables['Build.Reason'], 'PullRequest')
-  steps:
-
-  # Install .NET Core sdk
-  - task: UseDotNet@2
-    displayName: 'Install .NET Core sdk $(dotnetVersion)'
-    inputs:
-      packageType: sdk
-      version: $(dotnetVersion)
```

Comment:
do we still need to run the tests with the specific dotnet version?

Reply:
yes on azure pipeline